app/package/models.py

```diff
@@ -335,7 +340,8 @@ class MetaDataDB(db.Model):
     name = db.Column(db.TEXT, index=True)
     version = db.Column(db.TEXT, index=True, default='latest')
     descriptor = db.Column(db.JSON)
-    status = db.Column(db.TEXT, index=True, default='active')
+    status = db.Column(db.Enum(PackageStateEnum, native_enum=False),
+                       index=True, default=PackageStateEnum.active)
```

Comment:
In this case it will actually hurt performance. The index only makes sense when the value diversity is high, otherwise the search with the index will actually be slower than full-scan. I can't find good concise explanation online right now, but you can understand it if you learn how b-tree works https://en.wikipedia.org/wiki/B-tree.